staging/src/k8s.io/kubectl/pkg/cmd/config/view.go

```diff
@@ -136,6 +134,12 @@ func (o ViewOptions) Run() error {
 		return err
 	}
 
+	if !o.RawByteData {
```

Comment:
I would expect this as the last possible step before printing.

Reply:
Hrm, I'm not sure I understand this. I don't think the ordering should matter really, but I just stuck this at the front because I figured it would be best to redact sensitive info as soon as possible.

Reply:
@mpuckett159 what David wants is to move this down right before `return o.PrintObject(convertedObj, o.Out)`, this way all the minifying and flattening will always take place, and most importantly conversion to external scheme. 
This is good point, i haven't considered before.

Reply:
Apologies, I think I'm still missing something. The helper I wrote specifically works on the clientcmdapi Config object so that it can make use of the datapolicy tags to know what to redact without relying on manually adding keys to be redacted to the functions. I don't think that option would be available when working on the runtime Object interface. I could just cast it back to a Config I assume, but that feels wrong because I'd have to cast it back to a runtime Object before returning again.

Reply:
Just looked through the code again, and see where the confusion comes from, it's fine to use your method as is before the conversion and printing, so I'd suggest to ideally put it here: https://github.com/kubernetes/kubernetes/blob/9b9a963a8d0750735e2e6bbe4af5d1bb7cfc7f02/staging/src/k8s.io/kubectl/pkg/cmd/config/view.go#L152-L154